demo/webcam_demo.py

```diff
@@ -21,6 +21,11 @@
     'PyAVDecode', 'RawFrameDecode', 'FrameSelector'
 ]
 
+NEED_STUFF = [
+    'ResNetTSM', 'ResNetTIN', 'ResNet3dCSN', 'ResNet3d', 'ResNet2Plus1d',
```

Comment:
can this "stuff" thing be avoided completely?

Reply:
It can be avoided if we "stuff" for all models including `TSN`.

Reply:
so why not